core/src/test/java/cz/o2/proxima/util/TestUtils.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017-2020 O2 Czech Republic, a.s.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+
+import static org.junit.Assert.assertTrue;
+
+public class TestUtils {
+  /**
+   * Check if object is serializable
+   *
+   * @param object Object to check
+   * @throws IOException
+   */
+  public static void assertSerializable(Object object) throws IOException {
```

Comment:
This is cool. Can you reuse this please on other tests that test this? 
 - `StreamElementCoderTest.testCoderSerializable`
 - `ConfigRepositoryTest.testRepositoryNotSerializable` (actually tests the opposite, but that should not be a problem)
 - `DirectDataOperatorTest.testRepositoryNotSerializable` (dtto)
 - `cz.o2.proxima.direct.hbase.SerializationTest`
 - `cz.o2.proxima.direct.http.SerializableTest`

The modifications should be straightforward, if not, would you create issue to track it?

Reply:
fixed